app/connectors/HmrcTierConnectorWrapped.scala

```diff
@@ -20,25 +20,24 @@ import javax.inject.Inject
 import play.api.libs.json.{JsValue, Json}
 import play.api.{Configuration, Logger}
 import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
-import uk.gov.hmrc.play.bootstrap.http.HttpClient
-
+import uk.gov.hmrc.http.HttpClient
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 class HmrcTierConnectorWrapped @Inject()(val http: HttpClient, configuration: Configuration) {
 
   val serviceOriginatorIdKey: String = configuration.get[String]("microservice.services.nps.originatoridkey")
   val serviceOriginatorId: String = configuration.get[String]("microservice.services.nps.originatoridvalue")
+  private val logger = Logger(this.getClass)
```

Comment:
Can extend play.api.Logging which does the same thing and makes things a bit cleaner